webapp/src/components/workspaceSwitcher/workspaceOptions.tsx

```diff
@@ -102,7 +103,13 @@ const Option = (props: any): JSX.Element => {
             {
                 props.data.value !== DashboardOption.value &&
                 <div className='boardCount'>
-                    {props.data.boardCount} {props.data.boardCount > 1 ? 'Boards' : 'Board'}
+                    {
+                        intl.formatMessage({
```

Comment:
Why do you use `intl.formatMessage` instead of the `FormattedMessage` component?

Reply:
No specific reason. Why should I use `FormattedMessage` instead of `intl.formatMessage`? Is there any objective advantage in this particular case?

Reply:
The main reason is, you are representing a block here, so... using FormattedMessage looks like the right thing, probably also it is more perfomant because you don't need to re-render every time. Also, you don't need to use the `useIntl()` there.

Reply:
Done.

Also, this is such an insignificant change as it wasn't worth the effort to do this. Let just accept what's in place and be done with it.